packages/commonwealth/client/styles/components/component_kit/cw_tag.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../shared';
 
-.Tag {
+.Tag.Old {
```

Comment:
Let's remove `component_kit/cw_tag.scss` along with `cw_tag.tsx`. Old tags are not used anymore so we do not need them in the codebase

Reply:
Done!